### ISSUE TITLE
Possibly wrong batch norm behaviour at inference time

### ISSUE THREAD

**Issue by michaelaerni** [michaelaerni/ip6-molanet#20]
https://github.com/michaelaerni/ip6-molanet/blob/145d1f9584a9bf1ac11a0386a08106ac755ce720/molanet/operations.py#L160

We never never set the batch norm's `is_training` parameter to `False` during inference, probably leading to worse results as documented in https://www.tensorflow.org/versions/r1.2/api_docs/python/tf/contrib/layers/batch_norm

Furthermore, it might be the case that the variables are never updated which is also stated in the documentation.
